container-se-managed/src/main/java/org/jboss/arquillian/container/se/managed/ManagedSEContainerConfiguration.java

```diff
@@ -30,10 +30,19 @@
     private Level logLevel = Level.INFO;
     private boolean keepDeploymentArchives = false;
     private String additionalJavaOpts;
+    private int waitTime  = 5;
 
     public void validate() throws ConfigurationException {
     }
 
+    public void setWaitTime(int waitTime) {
+        this.waitTime = waitTime > 0 ? waitTime : 5;
```

Comment:
Well, the reason for this was to avoid nonsense wait times ( <=0) and to preserve default value should such wait times appear. But now that I see it, I was probably way too paranoid. Want me to remove it?